src/locales/es/messages.po

```diff
@@ -1425,7 +1425,7 @@ msgstr "esta interfaz"
 
 #: src/components/Create/RestrictedActionsForm.tsx
 msgid "ticketPrintingIsAllowed"
-msgstr "ticketPrintingIsAllowed"
+msgstr "ImprimirTicketsPermitido"
```

Comment:
@johnnyd-eth if we merge this before your other PR, think this will break anything in the app?